keyboards/dztech/dz60rgb/keymaps/default/config.h

```diff
@@ -0,0 +1,4 @@
+#pragma once
+#include "../../config.h"
```

Comment:
This line is not actually needed, and should be removed. 

```suggestion
```

Specifically, the build script automatically includes all of the config.h files, so this is not necessary.